doc/api/n-api.md

```diff
@@ -3083,6 +3085,26 @@ method or accessor, then the `this` argument to the callback is the wrapper
 object; the wrapped C++ instance that is the target of the call can be obtained
 then by calling `napi_unwrap()` on the wrapper object.
 
+### *napi_remove_wrap*
+<!-- YAML
+added: REPLACEME
+-->
+```C
+napi_status napi_remove_wrap(napi_env env,
+                             napi_value js_object,
+                             void** result);
+```
+
+ - `[in] env`: The environment that the API is invoked under.
+ - `[in] js_object`: The object associated with the native instance.
+ - `[out] result`: Pointer to the wrapped native instance.
```

Comment:
@mhdawson wanted an explicit `napi_remove_wrap()`, but, I guess an explicit `true`/`false` parameter is also, well, explicit.

If we do this within `napi_unwrap()`, though, we're treading into semver-major territory - which I OK while in experimental, but still, to be discussed, right?